### ISSUE TITLE
Weblogic 14c SPB 33452377 cannot be applied

### ISSUE THREAD

**Issue by dimovelev** [oracle/weblogic-image-tool#328]
I am trying to build an image for WebLogic 14.1.1.0.0 with the SPB https://updates.oracle.com/Orion/Services/download?type=readme&aru=24455391 and Oracle JDK 11.0.12. However, the build fails with the following error:
```
#19 [wls_build  9/10] RUN /u01/oracle/OPatch/opatch napply -silent -oh /u01/...
#19 3.189 Oracle Interim Patch Installer version 13.9.4.2.6
#19 3.189 Copyright (c) 2021, Oracle Corporation.  All rights reserved.
#19 3.189
#19 3.296
#19 3.296 Oracle Home       : /u01/oracle
#19 3.296 Central Inventory : /u01/oracle/oraInventory
#19 3.296    from           : /u01/oracle/oraInst.loc
#19 3.296 OPatch version    : 13.9.4.2.6
#19 3.296 OUI version       : 13.9.4.0.0
#19 3.296 Log file location : /u01/oracle/cfgtoollogs/opatch/opatch2021-12-07_17-28-51PM_1.log
#19 3.296
#19 3.296
#19 3.296 OPatch detects the Middleware Home as "/u01/oracle"
#19 3.296
#19 4.026 Verifying environment and performing prerequisite checks...
#19 7.427 UtilSession failed: The patch type is invalid in patch location: /u01/oracle/.patch_storage/patch_unzip/WLS_SPB_14.1.1.0.211009/
#19 7.602 Log file location: /u01/oracle/cfgtoollogs/opatch/opatch2021-12-07_17-28-51PM_1.log
#19 7.941
#19 7.941 OPatch failed with error code 73
#19 ERROR: executor failed running [/bin/sh -c /u01/oracle/OPatch/opatch napply -silent -oh /u01/oracle -nonrollbackable -phBaseDir /tmp/imagetool/patches         && test $? -eq 0         && /u01/oracle/OPatch/opatch util cleanup -silent -oh /u01/oracle]: exit code: 73
------
 > [wls_build  9/10] RUN /u01/oracle/OPatch/opatch napply -silent -oh /u01/oracle -nonrollbackable -phBaseDir /tmp/imagetool/patches         && test $? -eq 0         && /u01/oracle/OPatch/opatch util cleanup -silent -oh /u01/oracle:
------
executor failed running [/bin/sh -c /u01/oracle/OPatch/opatch napply -silent -oh /u01/oracle -nonrollbackable -phBaseDir /tmp/imagetool/patches         && test $? -eq 0         && /u01/oracle/OPatch/opatch util cleanup -silent -oh /u01/oracle]: exit code: 73
```

I also noticed that if I try to install patch p33452377_141100_Generic.zip somehow the imagetool also needs p28186730_139426_Generic.zip (that is not the case when I install the patch manually).

The command to build the image is the following:
```
imagetool/bin/imagetool.sh create --type WLSDEV --jdkVersion 11.0.12 --version 14.1.1.0.0 --tag core-weblogic-dev:14.1.1.0.0-SNAPSHOT --fromImage pb-container-registry.intra.loyaltypartner.com/ubi8/ubi:8.5-200 --patches 33452377 --additionalBuildCommands additional-commands.txt --additionalBuildFiles weblogic-deploy.tar.gz,docker-entry-point.sh --skipcleanup
```
I am not sure if that is a bug in the patch automation (does not work with SPBs) or I am doing something wrong.

**Comment by mriccell** [comment]
SPBs are not supported by the Image Tool because an SPB is not a patch but a mechanism similar to the Image Tool to apply all PSU and recommended CPU, and SPU patches to a WebLogic installation.
The SPB is equivalent to invoking Image Tool Create with --recommendedPatches option.

**Comment by dimovelev** [comment]
@mriccell thank you very much. i figured out from the SPB readme which patches it includes and applied them separately (with the --patches) parameter and it worked as expected. Maybe you could add a note somewhere that SPBs are not supported - or maybe that is obvious for people who work with patching all the time and it was just a rookie mistake. Nevertheless, I am closing this issue as it is not a bug but works as expected.

**Comment by ddsharpe** [comment]
@dimovelev Thank you for your feedback.  See fix #329, the documentation has been updated.
